processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -111,6 +112,17 @@ public BitmapFactory getBitmapFactory()
   }
 
   @Override
+  public BitmapIndex getBitmapIndex(String dimension)
```

Comment:
Should ```public BitmapIndex getBitmapIndex(String dimension)``` and ```   public ImmutableBitmap getBitmapIndex(String dimension, String value)``` be named differently? Maybe slightly confusing to have functions with a different return type with the same name